<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>977</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zamná</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>951</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(207, 226, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>271</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;Tipo de imágenes medicas a identificar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{

border: 1px solid gray;
border-color: #FF17365D;
border-radius: 15px;

}

QGroupBox::title{

border-top-left-radius: 9px;
border-top-right-radius: 9px;
padding: 5px 93 px;
background-color: #FF17365D;
color: rgb(255, 255, 255);


}</string>
     </property>
     <property name="title">
      <string>Tipo de imagen</string>
     </property>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Radiografías (tórax)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Fondo de ojo</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Mastografías</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Resonancia (Cerebro)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CT Tórax</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cantidad de imágenes que pueden ser detectadas.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Máximo de imágenes</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>321</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Opciones de visualización&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{

border: 1px solid gray;
border-color: #FF17365D;
border-radius: 15px;

}

QGroupBox::title{

border-top-left-radius: 9px;
border-top-right-radius: 9px;
padding: 5px 103px;
background-color: #FF17365D;
color: rgb(255, 255, 255);

}</string>
     </property>
     <property name="title">
      <string>Visualización</string>
     </property>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Diagnóstico</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Segmentación</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>1001</width>
      <height>951</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 251, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>981</width>
       <height>511</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{

border: 1px solid gray;
border-color: #FF17365D;
border-radius: 15px;

}

QGroupBox::title{

border-top-left-radius: 9px;
border-top-right-radius: 9px;
padding: 5px 455 px;
background-color: #FF17365D;
color: rgb(255, 255, 255);


}</string>
     </property>
     <property name="title">
      <string>Imagen</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>541</width>
        <height>461</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Imagen encontrada por el visor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>No se ha detectado ninguna imagen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>981</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{

border: 1px solid gray;
border-color: #FF17365D;
border-radius: 15px;

}

QGroupBox::title{

border-top-left-radius: 9px;
border-top-right-radius: 9px;
padding: 5px 455 px;
background-color: #FF17365D;
color: rgb(255, 255, 255);


}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>961</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1341</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOpciones_2">
    <property name="title">
     <string>Opciones</string>
    </property>
    <widget class="QMenu" name="menuImagen">
     <property name="title">
      <string>Imagen...</string>
     </property>
     <addaction name="actionInvertir_imagen"/>
     <addaction name="actionColor_de_segmentaci_n_2"/>
    </widget>
    <widget class="QMenu" name="menuVisualizador">
     <property name="title">
      <string>Visualizador...</string>
     </property>
     <addaction name="actionColor_de_segmentaci_n"/>
     <addaction name="actionColor_de_detecci_n"/>
     <addaction name="actionMostrar_informaci_n_en_visualizador"/>
    </widget>
    <addaction name="menuImagen"/>
    <addaction name="menuVisualizador"/>
    <addaction name="actionHistograma"/>
   </widget>
   <addaction name="menuOpciones"/>
   <addaction name="menuOpciones_2"/>
  </widget>
  <action name="actionInvertir_imagen">
   <property name="text">
    <string>Invertir imagen</string>
   </property>
   <property name="toolTip">
    <string>Invierte la imagen mostrada en la pantalla principal</string>
   </property>
  </action>
  <action name="actionColor_de_segmentaci_n">
   <property name="text">
    <string>Color de segmentación...</string>
   </property>
  </action>
  <action name="actionColor_de_detecci_n">
   <property name="text">
    <string>Color de detección...</string>
   </property>
  </action>
  <action name="actionS">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sí</string>
   </property>
  </action>
  <action name="actionNO">
   <property name="text">
    <string>No</string>
   </property>
  </action>
  <action name="actionMostrar_informaci_n_en_visualizador">
   <property name="text">
    <string>Mostrar información en visualizador</string>
   </property>
  </action>
  <action name="actionColor_de_segmentaci_n_2">
   <property name="text">
    <string>Color de segmentación...</string>
   </property>
  </action>
  <action name="actionHistograma">
   <property name="text">
    <string>Histograma</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
